asdf.sh

```diff
@@ -17,8 +17,10 @@ ASDF_DIR="$(dirname "$current_script_path")"
 # replace all occurrences - ${parameter//pattern/string}
 ASDF_BIN="${ASDF_DIR}/bin"
 ASDF_SHIMS="${ASDF_DIR}/shims"
+ASDF_USER_SHIMS="${HOME}/.asdf/shims"
```

Comment:
If we switch to pulling from an environment variable we will need to update this code as well. Something along the lines of:

```
ASDF_USER_SHIMS="${ASDF_DATA_DIR:-$HOME/.asdf}"
```